core/config/toml/types.go

```diff
@@ -34,10 +34,11 @@ var ErrUnsupported = errors.New("unsupported with config v2")
 // Core holds the core configuration. See chainlink.Config for more information.
 type Core struct {
 	// General/misc
-	AppID               uuid.UUID `toml:"-"` // random or test
-	InsecureFastScrypt  *bool
-	RootDir             *string
-	ShutdownGracePeriod *commonconfig.Duration
+	AppID                    uuid.UUID `toml:"-"` // random or test
+	InsecureFastScrypt       *bool
+	RootDir                  *string
+	ShutdownGracePeriod      *commonconfig.Duration
+	FeedsManagerSyncInterval *commonconfig.Duration
```

Comment:
New config option was added at the root level because I couldn't find a better place. Happy to move it elsewhere as per the maintainers advice.